python_docs_theme/static/search-focus.js

```diff
@@ -0,0 +1,10 @@
+document.addEventListener('keydown', function(event) {
+  if (event.key === '/') {
```

Comment:
Should this also check that the search box is not already focused?  Currently I can't enter `/` in the search box.

Reply:
Yes, updated.